projects/data-example-app/src/state/story.selectors.ts

```diff
@@ -1,7 +1,7 @@
 import { createSelector } from '@ngrx/store';
 import { Stories, Story } from './story';
 
-export const selectStories = createSelector<Story[], Story[], Story[][]>(
+export const selectStories = createSelector<Story[], [Story[]], Story[][]>(
```

Comment:
Explicitly provided generic arguments must take the format `<State, [S1, S2, Etc], Result>`